lib/videojs5-hlsjs-source-handler.js

```diff
@@ -52,6 +52,22 @@ var attachVideojsStreamrootProvider = function (window, videojs, Hls) {
 
         function load(source) {
             _hls.loadSource(source.src);
+
+            if (!_hls.config.autoStartLoad) {
+                var checkStartLoad = function(e) {
```

Comment:
i would probably put everything inside this if block into another function outside of `load` called something like `registerStartLoadOnPlay`. that would improve readability.